microsoft/PowerToys
# Change behavior on Edit Keyboard screen to be physical keys

## Summary of the Pull Request
This PR makes the following changes:
- On the Edit Keyboard screen, key remappings are not applied, so even if the user has orphaned a key, then can still search with it in the drop down menu
- On the Edit Shortcuts screen, shortcut remappings are not applied, so even if the user has orphaned a shortcut, then can still use it on that window. This is for consistency with the above behavior.
- Commented out App-Specific and ToggleKeyToModifier handlers in the back-end (even though they wouldn't technically do anything unless they were in the settings). This can be uncommented when we add in the feature to the UI.


## PR Checklist
* [X] Applies to #2551 


## Validation Steps Performed
Manually verified that even if a key is orphaned you can
- Still use it in Edit Keyboard
- You can't use it if you change focus to another app and try typing - To make sure "Apply" makes sense
